src/board.cpp

```diff
@@ -0,0 +1,166 @@
+#include <iostream>
+#include <vector>
+#include <string>
+#include "board.h"
+#include "gui.h"
+
+Board::Board() : Board(9, false) {}
+
+Board::Board(size_t size, bool cpu) {
+  isCpu = cpu;
+  p1Score = 0;
+  p2Score = 0;
+  boardSize = size;
+  for (int i = 1; i <= size; i++) {
+    p1.insert(i);
+    p2.insert(i);
+  }
+}
+
+int Board::move(int player, int number) {
+  std::set<int>& playerSet = player == 1 ? p1 : p2;
```

Comment:
```suggestion
  std::set<int>& playerSet = ((player == 1) ? p1 : p2);
```
at the very least, parens around the inner

Reply:
Should this be ordered?

Reply:
Hmm, no?? I don't see a real benefit. Perhaps it might be helpful for printing all player moves, but I convert the set to a boolean array vector anyways...

Reply:
then make it an `unordered_set`